packages/manager/modules/veeam-enterprise/src/details/routing.js

```diff
@@ -25,15 +25,22 @@ export default /* @ngInject */ ($stateProvider) => {
         goToLicenseTerminate: /* @ngInject */ ($state) => () =>
           $state.go('veeam-enterprise.details.dashboard.license.terminate'),
         breadcrumb: /* @ngInject */ (serviceName) => serviceName,
+        trackingPrefix: () => {
+          return TRACKING_PREFIX;
+        },
+        trackClick: /* @ngInject */ (atInternet, trackingPrefix) => (hit) => {
+          atInternet.trackClick({
+            name: `${trackingPrefix}::${hit}`,
```

Comment:
IDK if `trackingPrefix` is injected elsewhere. But if not, no need to make a resolve out of the `TRACKING_PREFIX` constant
```suggestion
        trackClick: /* @ngInject */ (atInternet) => (hit) => {
          atInternet.trackClick({
            name: `${TRACKING_PREFIX}::${hit}`,
```

Reply:
Indeed it is not necessary, I have modified